common/djangoapps/course_modes/migrations/0005_auto_20151217_0958.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('course_modes', '0004_auto_20151113_1457'),
+    ]
+
+    operations = [
+        migrations.RemoveField(
```

Comment:
Has the code change to read _expiration_datetime already been pushed? If not, I'd be concerned if dropping a column is not going to be compatible and will be a problem when migrations are applied ahead of code refresh. Also, do we need to migrate data from one column to another?   Is there a way to do a COLUMN rename instead?